src/cocaine/burlak/metrics/procfs.py

```diff
@@ -275,3 +292,176 @@ def calc_load(total, used, cached):
             used = used - cached
 
         return clamp(used / float(total), .0, 1.0)
+
+
+class IfSpeed(ProcfsMetric):
+    """Read NIC speed from sysfs.
+
+    TODO(IfSpeed): Deprecated: it seems that there is no any portable way
```

Comment:
It is weird to add a deprecated code. Is it needed?

Reply:
It really not yet deprecated, but likely to became soon, it is stub to gather run-time feedback for some probably short period, after which it would be decided either to retire it or not.